x11-libs/gdk-pixbuf/gdk-pixbuf-2.40.0-r1.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+# TODO: EAPI-7 needs gnome2-utils fixes - ${EROOT%/} and co usage for EAPI-7 in gdk_pixbuf_*
+# TODO: functions and eutils inherit for emktemp or some other solution
+
+inherit gnome.org gnome2-utils meson multilib multilib-minimal xdg
+
+DESCRIPTION="Image loading library for GTK+"
+HOMEPAGE="https://git.gnome.org/browse/gdk-pixbuf"
+
+LICENSE="LGPL-2+"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="X gtk-doc +introspection jpeg tiff webp"
+
+# TODO: For windows/darwin support: shared-mime-info conditional, native_windows_loaders option review
+DEPEND="
+	>=dev-libs/glib-2.48.0:2[${MULTILIB_USEDEP}]
+	x11-misc/shared-mime-info
+	>=media-libs/libpng-1.4:0=[${MULTILIB_USEDEP}]
+	jpeg? ( virtual/jpeg:0=[${MULTILIB_USEDEP}] )
+	tiff? ( >=media-libs/tiff-3.9.2:0=[${MULTILIB_USEDEP}] )
+	X? ( x11-libs/libX11[${MULTILIB_USEDEP}] )
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+"
+# librsvg blocker is for the new pixbuf loader API, you lose icons otherwise
+RDEPEND="${DEPEND}
+	!<gnome-base/gail-1000
+	!<gnome-base/librsvg-2.31.0
+	!<x11-libs/gtk+-2.21.3:2
+	!<x11-libs/gtk+-2.90.4:3
+"
+DEPEND="
```

Comment:
so this is replacing the above `DEPEND` - I don't think you want that. If you have dependencies that are
1) `DEPEND` only
2) `RDEPEND` only

in the same ebuild, then put the ones that are common to both into `COMMON_DEPEND` - that would be your current first DEPEND block as far as I can see.

Reply:
Originally I didn't alter any parts of gdk-pixbuf since I'm not maintainer of this ebuild, but if no objection made, I'll adapt it :+1: